src/main/resources/language.yml

```diff
@@ -93,7 +93,7 @@ CannotGrantAccessToSelf: "&cYou cannot grant access to yourself."
 RightClickGrantAccess: "&eRight click a chest or door to grant #name# access. Type /mf grantaccess cancel to cancel this."
 AlertGrantedIndependence: "&a#name# has granted your faction independence."
 AlertNoLongerVassalFaction: "&c#name# is no longer a vassal faction."
-FactionHomeNotSetYet: "&cThe faction home isn't set yet".
```

Comment:
This was the cause of the startup error, if anyone is curious.

Reply:
Sorry about it. Can u use https://yamlchecker.com/ to check yml file if file have another typo?...

Reply:
No worries! It's valid 👍

![image](https://user-images.githubusercontent.com/21204351/190544406-2be32046-933c-42e9-8547-4a9a1ad54a96.png)